File: app/src/main/java/com/user/canopas/dagger/Api/ApiService.java
package com.user.canopas.dagger.Api;


import com.user.canopas.dagger.mvp.model.ActorRespone;


import retrofit2.http.GET;
import rx.Observable;

public interface ApiService {

    @GET("/JSONParsingTutorial/jsonActors")
     Observable<ActorRespone> getActors();
}

File: app/src/main/java/com/user/canopas/dagger/Api/ApiService.kt
package com.user.canopas.dagger.Api


import com.user.canopas.dagger.mvp.model.ActorRespone


import retrofit2.http.GET
import rx.Observable

interface ApiService {

    @get:GET("/JSONParsingTutorial/jsonActors")
    val actors: Observable<ActorRespone>
}
